src/app/pages/organization/sites/site/site-dialog.component.ts

```diff
@@ -1,21 +1,23 @@
 import { AfterViewInit, Component, Inject, ViewChild } from '@angular/core';
 import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
-import { DialogData } from 'types/Authorization';
+import { SiteDialogData } from 'types/Authorization';
 
 import { Utils } from '../../../../utils/Utils';
 import { SiteComponent } from './site.component';
 
 @Component({
-  template: '<app-site #appRef [currentSiteID]="siteID" [inDialog]="true" [dialogRef]="dialogRef"></app-site>',
+  template: '<app-site #appRef [currentSiteID]="siteID" [canUpdateSite]="canUpdateSite" [inDialog]="true" [dialogRef]="dialogRef"></app-site>',
```

Comment:
Should we pass the site so if we enrich again the object, we will have new props for free without touching template.